app/src/main/java/com/example/plantsapp/presentation/ui/plantdetail/PlantDetailFragment.kt

```diff
@@ -41,4 +47,16 @@ class PlantDetailFragment : Fragment(R.layout.fragment_plant_detail) {
             )
         }
     }
+
+    companion object {
+        private const val ARGUMENT_PLANT_NAME = "PLANT NAME"
```

Comment:
![image](https://user-images.githubusercontent.com/23121846/137927384-fd0f45c6-3109-433f-afde-e104da27ba85.png)
You can use something like this to simplify code